ui/components/app/flask/snap-authorship/snap-authorship.js

```diff
@@ -59,7 +59,7 @@ const SnapAuthorship = ({ snapId, className }) => {
       paddingRight={4}
       borderRadius={BorderRadius.pill}
       display={DISPLAY.FLEX}
-      style={{ maxWidth: 'fit-content', width: '100%' }}
+      style={{ width: '100%' }}
```

Comment:
We have added width prop to Box recently, I think we can use it here to remove the inline css
```suggestion
      width={BLOCK_SIZES.FULL}
```